### PR TITLE
Updated Readme.md

### PR DESCRIPTION
**Pull request by DanBrown47**

### CODE REVIEW

**Comment by GouravSardana** [comment]
@DanBrown47 Awesome. But just to be more precise, Please also add how to install dynamic-cli using pip.

**Comment by DanBrown47** [comment]
The readme has updated with the installation procedures!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Dynamic - CLI
->  A CLI utility that can connect to an external databases.
+>  A CLI utility that can connect to external databases.
```
Comment (kaustubhgupta) [inline README.md:2]: Either it should be :  
1. to an external database
2. to external databases.
image

**Comment by GouravSardana** [comment]
@DanBrown47 Remove the sudo from the readme. Without sudo it will work. Please add few more things.
- Add how to install pip3.
- Add how you can install dynamic-cli using pip like pip install -e <your path>
- Use pip3 instead of pip because python2 is depreciated.

**Pull request merged**